pygin/modules/pygin/far.py

```diff
@@ -445,28 +474,52 @@ class FileTime(datetime.datetime):
 	def __new__(self, Value = 0):
 		return datetime.datetime.utcfromtimestamp((max(Value, _EPOCH_AS_FILETIME) - _EPOCH_AS_FILETIME) / _HUNDREDS_OF_NANOSECONDS)
 
+	def value(self):
+		return int(self.replace(tzinfo=datetime.timezone.utc).timestamp() * _HUNDREDS_OF_NANOSECONDS + _EPOCH_AS_FILETIME)
+
 @unique
 class PluginPanelItemFlags(IntFlag):
 	Default                                         = 0
 	Selected                                        = bit(30)
 	ProcessDescription                              = bit(31)
 
+@unique
+class FileAttributeFlags(IntFlag):
+	Default					= 0
+	Readonly				= bit(0)
+	Hidden					= bit(1)
+	System					= bit(2)
+	Directory				= bit(4)
+	Archive					= bit(5)
+	Normal					= bit(7)
+	Temporary				= bit(8)
+	SparseFile				= bit(9)
+	ReparsePoint			= bit(10)
```

Comment:
Formatting here and below

Reply:
I guess the formatting in such cases will depend on the tab's size? What size should I use?

Reply:
Removed in 5ef1f612ff8787268b8d48658e3d1e8cad9b7272.